app/src/main/res/layout/fragment_product_detail.xml

```diff
@@ -262,35 +275,33 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_alignParentBottom="true"
-                android:background="@color/color_white"
-                android:orientation="horizontal"
-                android:padding="@dimen/dp_3"
+                android:background="@color/color_gray_white"
                 >
-                <ImageView
-                    android:id="@+id/image_user"
-                    android:layout_width="@dimen/dp_0"
-                    android:layout_height="@dimen/dp_45"
-                    android:layout_marginRight="@dimen/dp_3"
-                    android:layout_weight="1.3"
-                    android:padding="@dimen/dp_1"
-                    app:imageUrl="@{viewModel.userImage}"
-                    />
-                <EditText
-                    android:layout_width="@dimen/dp_0"
+                <LinearLayout
+                    android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:layout_weight="7"
-                    android:hint="@string/add_a_comment"
-                    android:text="@={viewModel.comment}"
-                    />
-                <ImageButton
-                    android:layout_width="@dimen/dp_0"
-                    android:layout_height="@dimen/dp_35"
-                    android:layout_marginLeft="@dimen/dp_3"
-                    android:layout_weight="1"
-                    android:background="@drawable/ic_send_gray"
-                    android:onClick="@{() -> viewModel.onClickSendComment()}"
-                    />
-
+                    android:layout_alignParentBottom="true"
```

Comment:
WARNING: [ObsoleteLayoutParam] Invalid layout param in a `LinearLayout`: `layout_alignParentBottom`
 The given layout_param is not defined for the given layout, meaning it has no effect. This usually happens when you change the parent layout or move view code around without updating the layout params. This will cause useless attribute processing at runtime, and is misleading for others reading the layout so the parameter should be removed.